eZ/Publish/Core/Persistence/Legacy/Content/Type/Handler.php

```diff
@@ -619,4 +619,23 @@ public function publish( $contentTypeId )
 
         $this->updateHandler->publishNewType( $toType, Type::STATUS_DEFINED );
     }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\Type\Handler::getFieldMap
+     */
+    public function getFieldMap()
+    {
+        $fieldMap = [];
+        $rows = $this->contentTypeGateway->getFieldMapData();
+
+        foreach ( $rows as $row )
+        {
+            $fieldMap[$row["content_type_identifier"]][$row["field_definition_identifier"]] = [
```

Comment:
Is it me, or we should try to make this a bit more readable ? We can create a couple temp variables, they'll get destroyed a couple lines later anyway. I'm mostly referring to the keys.